### ISSUE TITLE
Detach/attach content in onShown/onHidden

### ISSUE THREAD

**Issue by MiikaL** [lecar-red/bootstrapx-clickover#27]
Hi there,

I have a bit of a strange problem, and of course the culprit is IE9 (isn't it always?). I am using the clickover to show a navigational component. As I understand it, the clickover destroys itself whenever it is closed, so I am using the onShown and onHidden handlers along with jQuery .detach() and .append() commands to "save" the navigational component to an invisible div whenever the clickover is not visible.

This all works great on Firefox, Chrome, iOS Safari and even IE10. But in IE9, it seems that my navigational component is already destroyed by the time the onHidden handler is called. Would anybody know of any reason why this is, and what could be done to get around the problem?

**Comment by MiikaL** [comment]
Still having this problem, and I think the reason might be that IE9 doesn't support animations, so popover is destroyed before the onHidden handler is called, so there isn't a chance to "rescue" the content first.

However! What I would actually like is to be able to make a persistent popover, such that it is not destroyed at all between showings. This way I could just leave the content in place, and solve all problems.
